scripts/dev/recompile_requirements.py

```diff
@@ -161,8 +161,11 @@ def parse_args():
 
 def git_diff(*args):
     """Run a git diff command."""
-    command = (['git', '--no-pager', 'diff'] + list(args) + [
-        '--', 'requirements.txt', 'misc/requirements/requirements-*.txt'])
+    command = (
+        ["git", "--no-pager", "-c", "diff.mnemonicPrefix=false", "diff"]
```

Comment:
I had that config set locally, which causes git to use `w/` and `i/` for working dir and index, rather than `a/` and `b/` - but that broke the diff parsing the script does.